CONTRIBUTING.md

```diff
@@ -58,6 +58,54 @@ file.
   - Expand Defaults and verify that Flutter is present.
   - Click [+] and verify that Flutter is present.
 
+## Flutter plugin development on Windows
+
+These are notes taken while making a Windows dev env for the Flutter plugin.
+It assumes familiarity with the section about set-up on MacOS.
+However, this configuration includes IntelliJ source code.
+```bash
+Launch command-line tool: git-bash (may need to install -- don't remember)
```

Comment:
Is this git-bash? https://git-scm.com/downloads If so it probably does need to be installed.